### ISSUE TITLE
ios8.1 support needed

### ISSUE THREAD

**Issue by deepblue2000** [Bensge/NoAdstoreOpen#1]
cydia opens appstore @ios8.1 , so please an update

**Comment by Bensge** [comment]
[Cydia 1.1.14](https://cydia.saurik.com/info/cydia/) now blocks advertisements' force-opening of the AppStore automatically and shows a confirmation alert. Therefore this tweak is now obselete and there's no point in updating it to iOS 8 anymore.

**Comment by Alex793** [comment]
Yes but NoAdstoreOpen block also AppStore from Safari (many sites open AppStore with ads and with NAO we had the confirmation choice to block it), so we need an update for this.

Many thanks.

;)
